README.md

```diff
@@ -3,7 +3,7 @@
     <img src="http://i.imgur.com/RZbJy1u.png" alt="Hanzo" />
   </a>
   <br />
-  Hanzo is a sharp tool to handle deployments of multiple environments of the same application on Heroku.
+  Hanzo is a sharp tool to handle deploying an application on Heroku on multiple environments simultaneously.
```

Comment:
Hanzo itself doesn’t allow simultaneous deployment, you have to use `tmux` or else to achieve it.

Reply:
Eek!